Source/Charts/Data/Implementations/Standard/PieChartDataSet.swift

```diff
@@ -110,7 +110,9 @@ open class PieChartDataSet: ChartDataSet, IPieChartDataSet
 
     /// the color for the highlighted sector
     open var highlightColor: NSUIColor? = nil
-
+    
+    /// When onlyShowHighlightValue is true ,only show what you choose. default is false
+    open var onlyShowHighlightValue : Bool = false
```

Comment:
need a better name to reflect what's the property about.
Also, I think it should be a per chart view setting, as pie chart only has one data set, and it seems only show text value line makes more sense if it's for PieChartView itself?